### ISSUE TITLE
#-2094183267------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1841]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-name" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-b044e797-12a3-c53f-4243-66f2db623f7a">name</div>`

<strong>Element Location: </strong>#attr-label-name

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-2094183267_All page content should be contained by landmarks.png`
